detox/src/ios/expectTwo.js

```diff
@@ -10,6 +10,7 @@ const { assertEnum, assertNormalized } = require('../utils/assertArgument');
 const { removeMilliseconds } = require('../utils/dateUtils');
 const { actionDescription, expectDescription } = require('../utils/invocationTraceDescriptions');
 const log = require('../utils/logger').child({ cat: 'ws-client, ws' });
+const isRegExp = require('../utils/regexp');
```

Comment:
For consistency, I would suggest:

1) to rename the file to `utils/isRegExp.js`
2) use named exports, e.g.: `const { isRegExp } = require(...)`